neasqc_wp61/models/quantum/embedder.py

```diff
@@ -506,14 +522,19 @@ def _compute_embeddings(
         sentences = embeddings_df.sentence.values.tolist()
 
         if self.is_sentence_embedding:
-            vectorised_sentence_list = model.encode(sentences).tolist()
+            vectorised_sentence_array = model.encode(
+                sentences, convert_to_numpy=True
+            )
+            vectorised_sentence_list = [
+                array.flatten() for array in vectorised_sentence_array
+            ]
 
         else:
```

Comment:
dangerous leap of faith here. 
This else means basically anything else, so it could be a potatoe for all you know.
Have we made sure earlier on that this was checked?